template/partials/report.js

```diff
@@ -8,8 +8,12 @@ function datestuff (date) {
 }
 
 
-module.exports = function (results) {
+module.exports = function (results, compare) {
   const hx = hyperx(h)
+  return reportBody(results, hx, compare)
```

Comment:
before returning reportBody, remove any objects in compare that have the same finish and date as results, if compare exists and length is greater than 0